engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -74,12 +74,22 @@ import scala.concurrent.duration.DurationInt
 class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
 
   private val log = LoggerFactory.getLogger(this.getClass)
+  private val telemetryLog =
+    LoggerFactory.getLogger("org.enso.telemetry.languageserver.boot.MainModule")
   log.debug(
     "Initializing main module of the Language Server from [{}, {}, {}]",
     BuildVersion.currentEdition,
     serverConfig,
     logLevel
   )
+  telemetryLog.trace(
```

Comment:
Ideally this `telemetryLog` usage is merged with the above `log.debug`. 

```suggestion
  telemetryLog.debug(
```
and remove the line 79-84...

Reply:
Tried that:
```diff
diff --git a/engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala b/engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala
index 3be2c2d7b6..4a550e0011 100644
--- a/engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala
+++ b/engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala
@@ -76,19 +76,15 @@ class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
   private val log = LoggerFactory.getLogger(this.getClass)
   private val telemetryLog =
     LoggerFactory.getLogger("org.enso.telemetry.languageserver.boot.MainModule")
-  log.debug(
-    "Initializing main module of the Language Server from [{}, {}, {}]",
-    BuildVersion.currentEdition,
-    serverConfig,
-    logLevel
-  )
-  telemetryLog.trace(
-    "Initializing main module of the Language Server: edition={}, graal_version={}, enso_version={}, is_release={}, AOT={}",
+  telemetryLog.debug(
+    "Initializing main module of the Language Server: edition={}, graal_version={}, enso_version={}, is_release={}, AOT={}, logLevel={}, serverConfig={}",
     BuildVersion.currentEdition(),
     BuildVersion.graalVersion(),
     BuildVersion.ensoVersion(),
     BuildVersion.isRelease,
-    HostEnsoUtils.isAot
+    HostEnsoUtils.isAot,
+    logLevel,
+    serverConfig
   )
 
   private val contextSupervisor = new ComponentSupervisor()
```

but then, we have the following metadata in telemetry:
```json
    "metadata": {
      "type": "telemetry",
      "AOT": false,
      "edition": "0.0.0-dev",
      "enso_version": "0.0.0-dev",
      "graal_version": "24.0.0",
      "is_release": false,
      "logLevel": "DEBUG",
      "loggerName": "org.enso.telemetry.languageserver.boot.MainModule",
      "serverConfig": "LanguageServerConfig(127.0.0.1,60369,None,62761,None,175d856f-4415-4e70-b383-627c1d66f453,/home/pavel/Documents/enso-projects/NewProject7,903d9f40-7ca0-49b3-a7cd-b662d7203167,ProfilingConfig(None,None),StartupConfig(false),language-server,scala.concurrent.impl.ExecutionContextImpl$$anon$3@3f191845[Running, parallelism = 16, size = 0, active = 0, running = 0, steals = 0, tasks = 0, submissions = 0])"
    },
```
and `LanguageServerConfig` contains an absolute path. So let's not do that for now, and keep these two logs separate.